test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -4580,6 +4581,12 @@ public void PackCommand_PackLicense_SimpleExpression_StandardLicense(string lice
                     Assert.Equal(licenseMetadata.Version, LicenseMetadata.EmptyVersion);
                     Assert.Equal(licenseMetadata.License, licenseExpr);
                     Assert.Equal(licenseExpr, licenseMetadata.LicenseExpression.ToString());
+                    // Validate the nuspec round trips.
+                    using (var nuspecStream = nupkgReader.GetStream($"{packageName}.nuspec"))
+                    {
+                        var manifest = Packaging.Manifest.ReadFrom(nuspecStream, true);
```

Comment:
Would it make sense to run Validation on the nuspec produced in all of your pack tests?

Reply:
It does. But I'd have no way of ensuring that every future tests is validated like that. 

I'll add it to a bunch more tests and hope the future implementers/reviewers take the hint.